### DISCUSSION TITLE
Only tar file required?

### DISCUSSION THREAD

**Discussion by nick-traeger** [valhalla/valhalla#3750] [Q&A] [answered]
Hello, we would like to regularly renew or recreate the database on a second server and then copy it to the main server. Is it then enough if only the tar file created is copied? Or are also the folders "valhalla_tiles", "timezone_data" and "admin_data" necessary for operation?

**Comment by nilsnolde** [comment, marked as answer]
Only the tar file is enough. The rest is used when building the graph.

**Reply by markstos** [reply]
That's great to know!
